gobblin-api/src/main/java/gobblin/configuration/ConfigurationKeys.java

```diff
@@ -235,6 +242,13 @@
   public static final String DEFAULT_WRITER_FILE_PATH_TYPE = "default";
   public static final String SIMPLE_WRITER_DELIMITER = "simple.writer.delimiter";
   public static final String SIMPLE_WRITER_PREPEND_SIZE = "simple.writer.prepend.size";
+  public static final String WRITER_JDBC_INSERT_BATCH_SIZE = WRITER_PREFIX + ".jdbc.batch.size";
+  public static final int DEFAULT_WRITER_JDBC_INSERT_BATCH_SIZE = 30;
+  public static final String WRITER_JDBC_INSERT_BUFFER_SIZE = WRITER_PREFIX + ".jdbc.insert.buffer.size";
+  public static final int DEFAULT_WRITER_JDBC_INSERT_BUFFER_SIZE = 1024 * 1024; //1 MBytes
+  public static final int MAX_WRITER_JDBC_INSERT_BUFFER_SIZE = 10 * 1024 * 1024; //10 MBytes
+  public static final String WRITER_JDBC_MAX_PARAM_SIZE = WRITER_PREFIX + ".jdbc.insert.max.param.size";
```

Comment:
We try to avoid use `.` as a a replacement for spaces. So this config key should actually be `writer.jdbc.max_param_insert_size` (or something along those lines)